element_interface/utils.py

```diff
@@ -140,31 +145,57 @@ def ingest_csv_to_table(
         )
         if verbose:
             insert_len = len(table) - prev_len
-            print(
+            logger.info(
                 f"\n---- Inserting {insert_len} entry(s) "
                 + f"into {table.table_name} ----"
             )
 
 
-def recursive_search(key: str, dictionary: dict) -> any:
-    """Return value for key in a nested dictionary
-
-    Search through a nested dictionary for a key and returns its value.  If there are
-    more than one key with the same name at different depths, the algorithm returns the
-    value of the least nested key.
+def str_to_bool(value) -> bool:
+    """Return whether the provided string represents true. Otherwise false.
 
     Args:
-        key (str): Key used to search through a nested dictionary
-        dictionary (dict): Nested dictionary
+        value (any): Any input
```

Comment:
```suggestion
        value (str, bool, int): Any input
```

Reply:
Are there any other data types that would work?

Reply:
I think (str, bool, int) is exhaustive in terms of valid types. Other types would return False regardless of truthiness
```python
value_to_bool([True])
> False
```